### ISSUE TITLE
data-spa-component rendering template but not calling dataUrl

### ISSUE THREAD

**Issue by alxAgu** [sucom/SPA.js#9]
Hi, I'm using the new nested components structure (thanks heaps for that btw) and noticed that when importing components via html attribute (data-spa-component) the component's template gets rendered but the url described in the component's controller is never called.
```
// In app/components/user/profile/profile.html
< div data-spa-component="$/user/hobbies"></div>
```
```
// In app/components/user/hobbies/hobbies.js
spa.$('user/hobbies', {
   dataUrl: '/user/{id}/hobbies',
   dataUrlParams: 123,
   .... 
});
```

The html contents in app/components/user/hobbies/hobbies.html shows up but the API endpoint is never called. 

Calling spa.$render( "user/hobbies") works fine, rendering the template with the data received from the dataUrl endpoint.

Am I doing something wrong?

Thanks.

A.

P.S: Happy New Year! :-)

**Comment by sucom** [comment]
`<div data-spa-component="$user/hobbies"></div>` should work. There is no need for / before the root component. Please try and let me know. In the next update I will take care of it.

Thanks.

**Comment by sucom** [comment]
Sorry, I think I misunderstood your issue. Anyway, let me check and get back to you soon.

Thanks.

**Comment by alxAgu** [comment]
@sucom, you are absolutely right, the problem was the '/' at the beginning. Everything is working fine now. Thanks a lot.

A.

**Comment by sucom** [comment]
Awesome. Thanks for confirming.
Also, I fixed in code (v2.75.2) to take care of unnecessary '/' prefix.

Thanks again.
